README.md

```diff
@@ -12,29 +12,376 @@ This project introduces the verification of [Mina Protocol](https://minaprotocol
+  ```
+
+1. Set `contracts/script/deploy/config/holesky/aligned.holesky.config.json` to:
+
+  ```json
+  {
+    "chainInfo": {
+      "chainId": 17000
+    },
+    "permissions": {
+      "owner": "<operator_account_address>",
+      "aggregator": "<operator_account_address>",
+      "upgrader": "<operator_account_address>",
+      "churner": "<operator_account_address>",
+      "ejector": "<operator_account_address>",
+      "deployer": "<operator_account_address>",
+      "initalPausedStatus": 0
+    },
+    "minimumStakes": [
+      1
+    ],
+    "strategyWeights": [
+      [
+        {
+          "0_strategy": "0x80528D6e9A2BAbFc766965E0E26d5aB08D9CFaF9",
+          "1_multiplier": 1e+18
+        }
+      ]
+    ],
+    "operatorSetParams": [
+      {
+        "0_maxOperatorCount": 200,
+        "1_kickBIPsOfOperatorStake": 11000,
+        "2_kickBIPsOfTotalStake": 50
+      }
+    ],
+    "uri": ""
+  }
+  ```
+
+1. Setup the `contracts/scripts/.env` file. A template is available in `contracts/scripts/.env.example.holesky`. Set `PRIVATE_KEY` to the private key of the account you chose to fund the operator (the one with address `operator_account_address`).
+1. Set `contracts/script/output/holesky/alignedlayer_deployment_output.json` to:
+
+  ```json
+  {
+    "addresses": {
+      "alignedLayerProxyAdmin": "0x715e3F1c9F58832a606cf85C328250af32d20E93",
+      "alignedLayerServiceManager": "0x4d879A997f422FeB8AC2f3d9Dcc749e9d8d0E4a4",
+      "alignedLayerServiceManagerImplementation": "0x63257B1e36B78cb8be08A5A42317B93D7127D56b",
+      "blsApkRegistry": "0xf1e92D804230Cdb88D95A79e8D6E5B4e3182E632",
+      "blsApkRegistryImplementation": "0x6d3Bf64adAd29949766417543eF2eb8670964426",
+      "indexRegistry": "0xd2062Fb412923bE70F4d51E361Fe17408eE4b1b0",
+      "indexRegistryImplementation": "0x5Ee29B5d21CDA0ac24645Cb25DcdB9759E89628B",
+      "operatorStateRetriever": "0x38a45de56c29a3929Bb68aFD51d4B28F158d2900",
+      "registryCoordinator": "0x5EA37f0FdF065c047cdC0a5Ee7b3b99aC16C2B4a",
+      "registryCoordinatorImplementation": "0xE3Cef87B26f1A4CA37aF548e058f660F22Ef2E76",
+      "serviceManagerRouter": "0x6aA8AbaC32500eab9b67D4259602535619A6B919",
+      "stakeRegistry": "0x1763e9cEC4137a2031985A5D813dF5dC36ED71Ff",
+      "stakeRegistryImplementation": "0xd4a7d36B2c049613BEb5Ade3d92Af82aFc4a12d6"
+    },
+    "chainInfo": {
+      "chainId": 17000,
+      "deploymentBlock": 2421617
+    },
+    "permissions": {
+      "alignedLayerAggregator": "<operator_account_address>",
+      "alignedLayerChurner": "<operator_account_address>",
+      "alignedLayerEjector": "<operator_account_address>",
+      "alignedLayerOwner": "<operator_account_address>",
+      "alignedLayerUpgrader": "<operator_account_address>",
+      "pauserRegistry": "0x85Ef7299F8311B25642679edBF02B62FA2212F06"
+    }
+  }
+  ```

```

Comment:
Should the user deploy these contracts? (And in that case the addresses wouldn't be fixed) Because I don't remember if you needed to have the private key with which you deployed these contracts. Will check that.